pages/hub/lists/index.tsx

```diff
@@ -66,6 +115,59 @@ const ListsHub: WithPageLayout = () => {
           goToPage
         />
       </div>
+
+      <Dialog open={isDeleteOpen}>
```

Comment:
This doesn't block the PR, but I've seen the Dialog component being used in several places where we add the `<DialogContent />`, <DialogTitle />` etc.

This can be abstracted into a component that would look something like this.

```jsx
import { Dialog as RadixDialog, DialogContent, DialogDescription, DialogTitle } from "@radix-ui/react-dialog";
import { DialogCloseButton, DialogHeader } from "components/molecules/Dialog/dialog";

// define the types for the props for the DialogyComponent
interface DialogProps {
  children: React.ReactNode;
  title: string;
  description?: string;
  onClose: () => void;
  onOpen: boolean;
  onOpenChange: (open: boolean) => void;
}

export function Dialog({ children, title, description, onClose, onOpen: isOpen, onOpenChange }: DialogProps) {
  return (
    <RadixDialog open={isOpen} onOpenChange={onOpenChange}>
      <DialogContent className="p-4">
        <DialogHeader>
          <DialogTitle>{title}</DialogTitle>
          {description && <DialogDescription className="font-normal">{description}</DialogDescription>}
        </DialogHeader>
        {children}
        <DialogCloseButton onClick={onClose} />
      </DialogContent>
    </RadixDialog>
  );
}
```


I'll create an issue for it.

Reply:
I created https://github.com/open-sauced/app/issues/1741